puppet/modules/consul/manifests/init.pp

```diff
@@ -120,7 +120,9 @@
   }
 
   # install airworthy if necessary
-  ensure_resource('class', '::airworthy', {})
+  if !defined(Class[::airworthy]) {
+    ensure_resource('class', '::airworthy', {})
+  }
```

Comment:
Where is that change coming from?

I prefer the old variant. It feels like doing the if not defined step twice